ocs_ci/ocs/uninstall.py

```diff
@@ -37,23 +36,96 @@ def remove_ocp_registry_from_ocs(platform):
 
     """
     image_registry_obj = ocp.OCP(
-        kind=constants.CONFIG, namespace=constants.OPENSHIFT_IMAGE_REGISTRY_NAMESPACE
+        kind=constants.IMAGE_REGISTRY_CONFIG, namespace=constants.OPENSHIFT_IMAGE_REGISTRY_NAMESPACE
     )
-    param_cmd = None
+    param_cmd1 = None
+    param_cmd2 = None
     if platform.lower() == constants.AWS_PLATFORM:
-        param_cmd = '[{"op": "remove", "path": "/spec/storage"}]'
+        param_cmd1 = '[{"op": "remove", "path": "/spec/storage"}]'
+        param_cmd2 = '[{"op": "remove", "path": "/status/storage"}]'
 
     elif platform.lower() == constants.VSPHERE_PLATFORM:
-        param_cmd = '[{"op": "replace", "path": "/spec/storage", "value": {"emptyDir": "{}"}}]'
+        param_cmd1 = '[{"op": "replace", "path": "/spec/storage", "value": {"emptyDir": "{}"}}]'
+        param_cmd2 = '[{"op": "replace", "path": "/status/storage", "value": {"emptyDir": "{}"}}]'
 
-    if param_cmd:
+    if param_cmd1:
         image_registry_obj.patch(
-            resource_name=constants.IMAGE_REGISTRY_RESOURCE_NAME, params=param_cmd, format_type='json'
+            params=param_cmd1, format_type='json'
+        )
+        image_registry_obj.patch(
+            params=param_cmd2, format_type='json'
         )
     else:
         log.info("platform registry not supported")
```

Comment:
```python
        params_list = list()
        if platform.lower() == constants.AWS_PLATFORM:
            params_list.apend('[{"op": "remove", "path": "/spec/storage"}]')
            params_list.apend('[{"op": "remove", "path": "/status/storage"}]')

        elif platform.lower() == constants.VSPHERE_PLATFORM:
            params_list.apend('[{"op": "replace", "path": "/spec/storage", "value": {"emptyDir": "{}"}}]')
            params_list.apend('[{"op": "replace", "path": "/status/storage", "value": {"emptyDir": "{}"}}]')

        if params_list:
            for params in params_list:
                image_registry_obj.patch(params=params, format_type='json')
        else:
            log.info("platform registry not supported")
```

Reply:
BTW, what about other platforms?

Reply:
new docs we have is only for azure. will implement with the 4.5 changes in a followup pr

Reply:
BM Platform missing  here

Reply:
@prsurve do you have the BM docs? I will add this platform too

Reply:
Vmware LSO changes will  be applicable for BM LSO

Reply:
@annagitel above comment is not addressed

Reply:
@annagitel my initial comment above is still not updated

Reply:
firs comment changes done, for the other platforms - will be implemented in a followup PR

Reply:
thanks :+1: